docs/reference/commandline/index.md

```diff
@@ -0,0 +1,85 @@
+<!-- [metadata]>
++++
+title = "Using the command line"
+description = "Docker's CLI command description and usage"
+keywords = ["Docker, Docker documentation, CLI,  command line"]
+[menu.main]
+identifier= "smn_cli"
+parent = "mn_reference"
++++
+<!-- [end-metadata]-->
+
+
+# The Docker commands
+
+This section contains reference information on using Docker's command line client. Each command has a reference page along with samples. If you are unfamiliar with the command line, you should start by reading about how to ["Use the Docker command line"](cli).  
+
+You start the Docker daemon with the command line. How you start the daemon affects your Docker containers. For that reason you should also make sure to read the [`daemon`](daemon) reference page.
+
+### Docker management commands
+
+* [daemon](daemon)
+* [info](info)
+* [inspect](inspect)
+* [version](version)
+
+### Image commands
+
+* [build](build)
+* [commit](commit)
+* [history](history)
+* [images](images)
+* [import](import)
+* [load](load)
+* [rmi](rmi)
+* [save](save)
+* [tag](tag)
+
+### Container commands
+
+* [attach](attach)
+* [cp](cp)
+* [create](create)
+* [diff](diff)
+* [events](events)
```

Comment:
events is probably "management"? not sure.

Was looking at the API docs what sections we used there: https://docs.docker.com/reference/api/docker_remote_api_v1.21/